src/kagglehub/logger.py

```diff
@@ -39,7 +39,7 @@ def _configure_logger(log_dir: Optional[Path] = None) -> None:
         handler = library_logger.handlers.pop()
         while handler.filters:
             handler.filters.pop()
-    logging_enabled = os.environ.get(KAGGLE_LOGGING_ENABLED_ENV_VAR_NAME, False)
+    logging_enabled = os.environ.get(KAGGLE_LOGGING_ENABLED_ENV_VAR_NAME, "False")
     if str(logging_enabled).lower() in ("true", 1):
```

Comment:
we won't need the string conversion now that we are returning a string.

Reply:
Yep, I remembered to make that change, too.